apps/web/components/availability/SkeletonLoader.tsx

```diff
@@ -1,8 +1,8 @@
+import classNames from "classnames";
```

Comment:
This diff is here since the v1 and v2 file had the same name, I moved the v2 to this place and deleted v1. So here you see the differences between these two 🙂

Reply:
i dont believe we need this package anymore since we have `@lib/classnames`

Reply:
Oh that's right indeed. I will already add that fix in this PR 👍